bitnami/seaweedfs/templates/master/statefulset.yaml

```diff
@@ -92,7 +92,9 @@ spec:
           args:
             - -ec
             - |
-              find {{ .Values.master.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }}
+              mkdir -p {{ .Values.master.persistence.mountPath }}
+              chown -R {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }} {{ .Values.master.persistence.mountPath }}
+              find {{ .Values.master.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs -r chown -R {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }}
```

Comment:
Could you follow the same approach that we currently have in the template? https://github.com/bitnami/charts/blob/c414d935017cac1c1d683115df2d919cf92c530c/template/CHART_NAME/templates/_init_containers.tpl#L29-L34